Documentation.app/Contents/MacOS/Quick start.playground/contents.xcplayground

```diff
@@ -3,6 +3,7 @@
     <pages>
         <page name='Installation'/>
         <page name='Creating an Xcode Playground'/>
+        <page name='Creating a Playground Book'/>
```

Comment:
Perhaps add `for iPad` at the end?

Reply:
Playground Book only works on iPads, for the title I prefer to keep it out (to be shorter) - and apply as you did in the description (features - readme)

Reply:
I'm not sure if people are familiar enough with the term Playground Book to understand that is iPad only... I still think the term iPad should appear on the side menu, at least to be more catchy.

Reply:
Ok, I'll change it